RecoParticleFlow/PFClusterProducer/plugins/CudaPFCommon.h

```diff
@@ -29,31 +29,32 @@ namespace PFClustering {
       float constantTerm2;
       float resHighE2;
 
-      TimeResConsts()
-          : corrTermLowE(-1.0),
-            threshLowE(-1.0),
-            noiseTerm(-1.0),
-            constantTermLowE2(-1.0),
-            noiseTermLowE(-1.0),
-            threshHighE(-1.0),
-            constantTerm2(-1.0),
-            resHighE2(-1.0){};
+     // TimeResConsts()
```

Comment:
ok. can you just remove these commented-out lines?

Reply:
Okay, I removed them.